website/server/controllers/top-level/dataexport.js

```diff
@@ -122,7 +156,7 @@ api.exportUserDataJson = {
  * @apiGroup DataExport
  * @apiDescription NOTE: Part of the private API that may change at any time.
  *
- * @apiSuccess {String} A xml file
+ * @apiSuccess {String} data A xml file. See /export/userdata.json for structure info.
```

Comment:
`a xml` -> `an xml`

It should probably stat that it's an xml file of the user object and link to the get user route.